third_party/git-cmake-format/CMakeLists.txt

```diff
@@ -1,16 +1,15 @@
 message(STATUS "Fetching git-cmake-format")
 include(FetchContent)
 FetchContent_Declare(
-    git_cmake_format
-    GIT_REPOSITORY https://github.com/ginkgo-project/git-cmake-format.git
-    GIT_TAG        e9a82f20d36f1a92d4c52910a8d0a694e7673c54
+        git_cmake_format
+        GIT_REPOSITORY https://github.com/ginkgo-project/git-cmake-format.git
+        GIT_TAG 966837ee83f2b3b7df9acf59e9d801fcd189dd9b
 )
 FetchContent_GetProperties(git_cmake_format)
 if(NOT git_cmake_format_POPULATED)
     FetchContent_Populate(git_cmake_format)
 
-    if(NOT DEFINED GCF_FORCE_OVERWRITE)
-        set(GCF_FORCE_OVERWRITE ON CACHE INTERNAL "If true, always overwrite pre-commit hook and script")
-    endif()
+    set(GCF_FORCE_OVERWRITE ON CACHE INTERNAL "If true, always overwrite pre-commit hook and script")
```

Comment:
does it need to be always on?

Reply:
If there's no GCF_FORCE_OVERWRITE before, it always sets it on.
It's off when user specifies it, right?

Reply:
You're, right. I will use `option` for this. It should be more concise.

Reply:
not sure whether you need to use an option for it. keeping it as the original develop looks good to me

Reply:
Option does exactly the same, but in 1 line instead of 3.

Reply:
I see